File: src/pedido.py
from datetime import date

class Pedido:
    def __init__(self, id_pedido: int, fecha: date, estado: str, empresa_transporte: str):
        self.__id_pedido = id_pedido
        self.__fecha = fecha
        self.__estado = estado
        self.__empresa_transporte = empresa_transporte

    def armar(self):
        self.__estado = "Armado"
        print("El pedido ha sido armado")

    def despachar(self):
        self.__estado = "Despachado"
        print("El pedido ha sido despachado con éxito")

# --- Getters ---
    def get_id_pedido(self) -> int:
        return self.__id_pedido

    def get_fecha(self) -> date:
        return self.__fecha

    def get_estado(self) -> str:
        return self.__estado

    def get_empresa_transporte(self) -> str:
        return self.__empresa_transporte

File: Televentas Java/Clases/Pedido.java
package Clases;

import java.time.LocalDate;

public class Pedido {

    private int idPedido;
    private LocalDate fecha;
    private String estado;
    private String empresaTransporte;

    public Pedido(int idPedido, LocalDate fecha, String estado, String empresaTransporte) {
        this.idPedido = idPedido;
        this.fecha = fecha;
        this.estado = estado;
        this.empresaTransporte = empresaTransporte;
    }

    public void armar() {
        this.estado = "Armado";
        System.out.println("El pedido ha sido armado");
    }

    public void despachar() {
        this.estado = "Despachado";
        System.out.println("El pedido ha sido despachado con éxito");
    }

    // --- Getters ---
    public int getIdPedido() { return idPedido; }
    public LocalDate getFecha() { return fecha; }
    public String getEstado() { return estado; }
    public String getEmpresaTransporte() { return empresaTransporte; }
}
